pkg/apis/cnsoperator/config/cnsregistervolume_crd.yaml

```diff
@@ -45,6 +47,7 @@ spec:
               description: DiskUrlPath is URL path to an existing block volume to be
                 imported into Project Pacific cluster.
               type: string
+              pattern: '^(?:http(s)?:\/\/)?[\w.-]+(?:\.[\w\.-]+)+[\w\-\._~:/?#[\]@!\$&'\(\)\*\+,;=.]+$'
```

Comment:
Can we change this to check the valid file path URL, not just validate any www URL?

for example: `(http[s]?:\/\/)?([^\/\s]+\/folder\/)(.*)`